tests/integration/test_credentials.py

```diff
@@ -72,7 +72,10 @@ def test_access_secret_vs_profile_code(self, credentials_cls):
         )
 
         credentials_cls.assert_called_with(
-            access_key='code', secret_key='code-secret', token=mock.ANY
+            access_key='code',
+            secret_key='code-secret',
+            token=mock.ANY,
+            account_id=mock.ANY,
```

Comment:
Similar to the assume role unit tests, this was needed since we don't conditionally send the token either even if it is none.